DDx/DDx/AppDelegate.m

```diff
@@ -59,12 +59,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
             return NO;
         }
     } 
-    else 
-    {
```

Comment:
Let's add documentation telling devs how to use and test both connection methods
We'll also need to update our test cases accordingly